languages/cpp2/src/test/java/de/jplag/cpp2/CppLanguageTest.java

```diff
@@ -0,0 +1,98 @@
+package de.jplag.cpp2;
+
+import java.util.Arrays;
+
+import de.jplag.testutils.LanguageModuleTest;
+import de.jplag.testutils.datacollector.TestDataCollector;
+import de.jplag.testutils.datacollector.TestSourceIgnoredLinesCollector;
+
+public class CppLanguageTest extends LanguageModuleTest {
```

Comment:
Javadoc, what is tested? At which granularity? few sentences can provide outsiders clarity here!